docs/administration/garbage-collection/_index.md

```diff
@@ -13,10 +13,15 @@ When you delete images from Harbor, space is not automatically freed up. You mus
 
     ![Garbage collection](../../img/garbage-collection.png)
 
-1. To delete untagged artifacts, select the check box **Delete Untagged Artifacts**. 
+1. To delete untagged artifacts, select the check box **Delete Untagged Artifacts**.
+1. To dry run garbage collection, click **DRY RUN**. 
 1. To run garbage collection immediately, click **GC Now**.
 
-When you run garbage collection, Harbor goes into read-only mode. All modifications to the registry are prohibited.
+As of v2.1.0, Harbor runs garbage collection uninterrupted so users can continue pushing, pulling, deleting artifacts.
+
+**DRY RUN** prints the blobs eligible for deletion, a rough estimation of free up space without removing any data. 
+
+To avoid damaging the uploading artifact, the garbage collection introduces a time windows(2 hours) to reserve the recent uploaded layers. Garbage collection does not sweep the manifest & blob files that have a timestamp in the time window.
```

Comment:
so this means all blobs uploaded are safe from gc for at least 2 hours?  I don't think we discussed this before, so just making sure on meaning here

Reply:
`so this means all blobs uploaded are safe from gc for at least 2 hours? `
yes, otherwise the gc will interrupt the `docker push`.

`I don't think we discussed this before, so just making sure on meaning here`
We discussed it in the proposal review.